bullet_train/app/views/account/memberships/_form.html.erb

```diff
@@ -1,42 +1,44 @@
-    <div class="grid grid-cols-1 gap-y gap-x sm:grid-cols-6">
-      <div class="sm:col-span-3">
-        <%= render 'shared/fields/text_field', form: form, method: :user_first_name %>
-      </div>
 
-      <div class="sm:col-span-3">
-        <%= render 'shared/fields/text_field', form: form, method: :user_last_name %>
+  <% with_field_settings form: form do %>
+    <% if membership.unclaimed? || membership.tombstone? %>
+      <div class="grid grid-cols-1 gap-y gap-x sm:grid-cols-6">
+        <div class="sm:col-span-3">
+          <%= render 'shared/fields/text_field', form: form, method: :user_first_name %>
+        </div>
+
+        <div class="sm:col-span-3">
+          <%= render 'shared/fields/text_field', form: form, method: :user_last_name %>
+        </div>
       </div>
-    </div>
-  <% end %>
+    <% end %>
 
-  <% if membership.tombstone? %>
-    <% if cloudinary_enabled? %>
-      <%= render 'shared/fields/cloudinary_image', form: form, method: :user_profile_photo_id %>
+    <% if membership.tombstone? %>
+      <% if cloudinary_enabled? %>
+        <%= render 'shared/fields/cloudinary_image', form: form, method: :user_profile_photo_id %>
+      <% end %>
     <% end %>
-  <% end %>
 
-  <%= hidden_field_tag 'membership[role_ids][]', nil %>
-
-  <% Membership.assignable_roles.each do |role| %>
-    <% if role.manageable_by?(current_membership.roles) %>
-      <div class="flex items-top">
-        <%= form.check_box :role_ids, {multiple: true, class: "h-4 w-4 text-primary-500 focus:ring-primary-500 border-slate-300 rounded mt-0.5"}, role.id, nil %>
-        <label for="membership_role_ids_<%= role.id %>" class="ml-2 block select-none">
-          <%= t('.grant_privileges_of', role_key: t(".fields.role_ids.options.#{role.key}.label")) %>
-          <div class="mt-0.5 text-slate-400 font-light leading-normal">
-            <%= t(".fields.role_ids.options.#{role.key}.description") %>
-          </div>
-        </label>
-      </div>
+    <%= hidden_field_tag 'membership[role_ids][]', nil %>
+
+    <% Membership.assignable_roles.each do |role| %>
+      <% if role.manageable_by?(current_membership.roles) %>
+        <div class="flex items-top">
+          <%= form.check_box :role_ids, {multiple: true, class: "h-4 w-4 text-primary-500 focus:ring-primary-500 border-slate-300 rounded mt-0.5"}, role.id, nil %>
+          <label for="membership_role_ids_<%= role.id %>" class="ml-2 block select-none">
+            <%= t('.grant_privileges_of', role_key: t(".fields.role_ids.options.#{role.key}.label")) %>
+            <div class="mt-0.5 text-slate-400 font-light leading-normal">
+              <%= t(".fields.role_ids.options.#{role.key}.description") %>
+            </div>
+          </label>
+        </div>
+      <% end %>
     <% end %>
-  <% end %>
 
-  <%= render "account/memberships/fields", membership: membership, form: form %>
-  <%# 🚅 super scaffolding will insert new fields above this line. %>
```

Comment:
I removed this line so that if someone ejects this file they won't end up with duplicate fields when they super scaffold new ones.